serialapi/controller_timeouts.go

```diff
@@ -0,0 +1,51 @@
+package serialapi
+
+import (
+	"errors"
+	"github.com/gozwave/gozw/frame"
+	"github.com/gozwave/gozw/protocol"
+	"github.com/gozwave/gozw/session"
+)
+
+type ControllerTimeouts struct {
+	AckTimeout  byte
+	ByteTimeout byte
+}
+
+const (
+	ackTimeout  = 0x96 // 0x95 = 150 -- multiply by 10 to get 1500ms
+	byteTimeout = 0x0F // 0x0F = 15 -- multiply by 10 to get 150ms
+
+	errUnableToSetControllerTimeouts = "unable to set controller timeouts"
+)
+
+func (s *Layer) SetControllerTimeouts() (*ControllerTimeouts, error) {
+	done := make(chan *frame.Frame)
+
+	request := &session.Request{
+		FunctionID: protocol.FnSerialAPIGetTimeouts,
```

Comment:
Don't you want this to be `FnSerialAPISetTimeouts`?

Reply:
Yeah, not sure how it ended up there as 'Get'